src/test/java/easytests/api/v1/controllers/QuizzesControllerTest.java

```diff
@@ -1,12 +1,144 @@
+import easytests.core.options.builder.QuizzesOptionsBuilder;
+import easytests.core.services.IssuesService;
+import easytests.core.services.QuizzesService;
+import easytests.support.JsonSupport;
+import easytests.support.QuizzesSupport;
+import easytests.support.TesteesSupport;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import static org.mockito.BDDMockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+
 /**
  * @author miron97
  */
+@Import({QuizzesMapper.class, SwaggerRequestValidationConfig.class})
+@RunWith(SpringRunner.class)
+@WebMvcTest(controllers = QuizzesController.class, secure = false)
 public class QuizzesControllerTest {
-    /**
-     * list
-     */
+    private static String id = "id";
+    private static String inviteCode = "inviteCode";
+    private static String codeExpired = "codeExpired";
+    private static String startedAt = "startedAt";
+    private static String finishedAt = "finishedAt";
+    private static String testee = "testee";
+    private static String firstName = "firstName";
+    private static String lastName = "lastName";
+    private static String surname = "surname";
+    private static String groupNumber = "groupNumber";
+    private static String quiz = "quiz";
+
+    @Autowired
+    private MockMvc mvc;
+
+    @MockBean
+    private QuizzesService quizzesService;
+
+    @MockBean
+    private IssuesService issuesService;
+
+    @MockBean
+    private AccessControlLayerServiceInterface acl;
+
+    @MockBean
+    private QuizzesOptionsBuilder quizzesOptionsBuilder;
+
+    private QuizzesSupport quizzesSupport = new QuizzesSupport();
+    private TesteesSupport testeesSupport = new TesteesSupport();
+
+
+    @Test
+    public void testListSuccess() throws Exception {
+        final List<QuizModelInterface> quizzesModels = new ArrayList<>();
+        IntStream.range(0, 3).forEach(idx -> {
+            final QuizModel quizModel = new QuizModel();
+            quizModel.map(this.quizzesSupport.getEntityFixtureMock(idx));
+
+            TesteeModelInterface testeeModel = new TesteeModel();
+            testeeModel.map(testeesSupport.getEntityFixtureMock(0));
+            testeeModel.setQuiz(quizModel);
+            quizModel.setTestee(testeeModel);
+
+            if (quizModel.getId().equals(1)) {
+                quizzesModels.add(quizModel);
```

Comment:
seems that you trying to create list, but than you filter it to 1 element.
make list consist of two items, one with testee included other without it

Reply:
Add nulls values (where possible) into test case.